staging/istio/Chart.yaml

```diff
@@ -13,5 +13,16 @@ sources:
 home: http://github.com/istio/istio
 maintainers:
   - name: goeldeepak
+  - name: bcmendoza
 engine: gotpl
 icon: https://istio.io/latest/favicons/android-192x192.png
+dependencies:
```

Comment:
Looking at past istio bumps seems like we instead of doing this we need to update `istio/charts/*`?

Reply:
This was simply an update to Helm 3’s addition of the dependencies field to follow convention.

Of the 3 subcharts, 2 are defined by us and not bundled with Istio. For the 3rd, grafana, we just copy over some configmaps. There were no updates to the configmaps, so no need to update the subcharts this time.